### ISSUE TITLE
The video stream problem

### ISSUE THREAD

**Issue by lfhlfh0108** [MoynaChen/Tello_ROS#4]
The video from camera is so unclear, has too much mosaic and green red stuff...
Could you please tell me if it is the problem of my tello? or you have the same situation?
Thank you

**Comment by MoynaChen** [comment]
Do you use the code all by this project? If you git clone project from anqixu, you should modify the way of  showing images in tello_driver_node.py

**Comment by lfhlfh0108** [comment]
Yes I have modified tello_driver_node.py into your version, but it does not work this time.
